### PR TITLE
Update backup-restore-ios.md

### PR DESCRIPTION
**Pull request by msbemba**
Added a note about backing up the authenticator app before reset.

### CODE REVIEW

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/memdocs/issues/2870

**Comment by PRMerger12** [comment]
@msbemba : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by yogkumgit** [comment]
@lenewsad Please review and Signoff this PR. Thanks

File: memdocs/intune/enrollment/backup-restore-ios.md [hunk 1]
```diff
@@ -37,6 +37,9 @@ You might have to back up and restore an Intune Automated Device Enrollment (ADE
 
 To back up and restore an iOS/iPadOS device, you must follow the Apple instructions:
 
+> [!NOTE] 
+> If you are using Microsoft authenticator App, it is important to back up the credentials and accounts before factory reset. For more information, visit [Back up and recover account credentials in the Authenticator app](https://support.microsoft.com/account-billing/back-up-and-recover-account-credentials-in-the-authenticator-app-bb939936-7a8d-4e88-bc43-49bc1a700a40#:~:text=The%20Microsoft%20Entra%20Authenticator%20app,or%20having%20to%20recreate%20accounts.)
```
Comment (JohanFreelancer9) [inline memdocs/intune/enrollment/backup-restore-ios.md:41]:
```suggestion
> If you are using Microsoft Authenticator app, it is important to back up the credentials and accounts before factory reset. For more information, visit [Back up and recover account credentials in the Authenticator app](https://support.microsoft.com/account-billing/back-up-and-recover-account-credentials-in-the-authenticator-app-bb939936-7a8d-4e88-bc43-49bc1a700a40#:~:text=The%20Microsoft%20Entra%20Authenticator%20app,or%20having%20to%20recreate%20accounts.)
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by PRMerger20** [comment]
@msbemba : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: memdocs/intune/enrollment/backup-restore-ios.md [hunk 2]
```diff
@@ -37,6 +37,9 @@ You might have to back up and restore an Intune Automated Device Enrollment (ADE
 
 To back up and restore an iOS/iPadOS device, you must follow the Apple instructions:
 
+> [!NOTE] 
+> If you are using Microsoft Authenticator app, it is important to back up the credentials and accounts before factory reset. For more information, visit [Back up and recover account credentials in the Authenticator app](https://support.microsoft.com/account-billing/back-up-and-recover-account-credentials-in-the-authenticator-app-bb939936-7a8d-4e88-bc43-49bc1a700a40#:~:text=The%20Microsoft%20Entra%20Authenticator%20app,or%20having%20to%20recreate%20accounts.)
```
Comment (JohanFreelancer9) [inline memdocs/intune/enrollment/backup-restore-ios.md:41]:
```suggestion
> If you are using Microsoft Authenticator app, it is important to back up the credentials and accounts before factory reset. For more information, visit [Back up and recover account credentials in the Authenticator app](https://support.microsoft.com/account-billing/back-up-and-recover-account-credentials-in-the-authenticator-app-bb939936-7a8d-4e88-bc43-49bc1a700a40#:~:text=The%20Microsoft%20Entra%20Authenticator%20app,or%20having%20to%20recreate%20accounts.).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Apologies @msbemba, one more, please.

**Comment by PRMerger4** [comment]
@msbemba : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by yogkumgit** [comment]
@JohanFreelancer9

**Comment by yogkumgit** [comment]
@lenewsad Please review and Signoff this PR. Thanks

**Comment by lenewsad** [comment]
#sign-off

**Comment by yogkumgit** [comment]
@tiburd Please merge this PR. Thanks

**Pull request merged**